packages/commonwealth/client/scripts/views/pages/Editor/Editor.scss

```diff
@@ -0,0 +1,102 @@
+@import '../../../../styles/shared';
+
+html,
+body,
+#root {
+  max-height: 100vh;
+  overflow: auto;
+}
+
+#root {
+  // this is needed for mobile layout
+  display: flex;
+}
+
+// disable misc buttons on mobile for now.
+#mava,
+.ReactQueryDevtools * {
+  display: none;
+}
+
+.mdxeditor {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+
+  // this is needed so that we can have a progress indicator on top of the main
+  // content.
+  position: relative;
+
+  .mdxeditor-toolbar {
+    // TODO this only shows up properly for the *mobile* editor not the desktop
+    // one.
```

Comment:
is this todo resolved? if yes, lets remove this comment

Reply:
The mobile/desktop editor things will be resolved in the next version ...